domains/kaushik-kashyap.is-cool.dev.json

```diff
@@ -0,0 +1,21 @@
+{
+    "description": "i am a college student.just made my first website",
+
+    "domain": "is-cool.dev",
+    "subdomain": "kaushik-kashyap",
+
+    "owner": {
+        "repo": "https://github.com/kaushik-kashyap/kaushik-kashyap.git",
+        "email": "kknl65789@gmail.com"
+    },
+
+    "record": {
+        
+        
+        "CNAME": "kaushik-kashyap.is-cool.dev",
```

Comment:
Same domain as you are requesting, invalid.

Reply:
oh sorry my bad  😅